package.json

```diff
@@ -9,7 +9,7 @@
     "changelog": "conventional-changelog --infile CHANGELOG.md --same-file --release-count 0 --preset angular",
     "lint": "tslint -c tslint.json src/**/*.ts",
     "premocha": "npm run compile",
-    "mocha": "mocha tests/*.ts tests/**/*.ts --require ts-node/register",
+    "mocha": "mocha tests/*.ts tests/factory/*.ts tests/operator/*.ts --require ts-node/register",
```

Comment:
I haven't written the extra operators, so compiling the tests fail, it can be reverted when the extra operators are present again.